openneuro-cli: Error: Cannot find module 'superagent'

```
me@christop:/mnt/c/Users/me$ npm install -g openneuro-cli@2.4.3
/home/me/miniconda3/bin/openneuro -> /home/me/miniconda3/lib/node_modules/openneuro-cli/src/index.js
+ openneuro-cli@2.4.3
updated 1 package in 18.081s
me@christop:/mnt/c/Users/me$ openneuro
file:///home/me/miniconda3/lib/node_modules/openneuro-cli/src/download.js:1
Error: Cannot find module 'superagent'
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
me@christop:/mnt/c/Users/me$ rm -rf ~/.openneuro
me@christop:/mnt/c/Users/me$ openneuro
file:///home/me/miniconda3/lib/node_modules/openneuro-cli/src/download.js:1
Error: Cannot find module 'superagent'
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
me@christop:/mnt/c/Users/me$ node --version
v9.11.1
```

--- a/packages/openneuro-cli/package.json
+++ b/packages/openneuro-cli/package.json
@@ -23,7 +23,8 @@
     "inquirer": "^5.2.0",
     "mkdirp": "^0.5.1",
     "moment": "^2.22.2",
-    "openneuro-client": "^2.6.0"
+    "openneuro-client": "^2.6.0",
+    "superagent": "^4.1.0"
   },
   "scripts": {
     "openneuro": "node src/index.js"

